gui/src/components/s3/account-management.vue

```diff
@@ -700,7 +700,8 @@ export default class CortxAccountManagement extends Vue {
     const res: any = await Api.getAll(apiRegister.s3_account);
     this.accountsList = res && res.data ? res.data.s3_accounts : [];
     this.s3Url = res.data && res.data.s3_urls ? res.data.s3_urls : [];
-    if (this.s3Url[0] === "http://None") {
+    
+    if (this.s3Url.length && this.s3Url[0] === "http://None") {
```

Comment:
please check for this.s3URL is undefined or not

Reply:
naming convetion not followed. s3URL should be used as isS3URL

Reply:
The check is already there at line 702.